pkg/controller/seed-controller-manager/mla/mla.go

```diff
@@ -73,5 +87,8 @@ func Add(
 	if err := newProjectReconciler(mgr, log, numWorkers, workerName, versions, grafanaClient); err != nil {
 		return fmt.Errorf("failed to create mla project controller: %v", err)
 	}
+	if err := newUserProjectBindingReconciler(mgr, log, numWorkers, workerName, versions, grafanaClient, &http.Client{}, grafanaURL, grafanaHeader); err != nil {
```

Comment:
http client here is not configured thus has 0 timeouts, this is really bad for anything.

Reply:
and it looks like that we could even remove those params (&http.Client{}, grafanaURL, grafanaHeader) as there is a standard method that can be used from sdk instead of custom calls.

Reply:
that not the method that we need. we need to do `post` instead of `get`